examples/inbox/package.json

```diff
@@ -11,6 +11,7 @@
   "license": "MIT",
   "dependencies": {
     "@babel/runtime": "^7.16.7",
+    "@cardinal/namespaces-components": "2.5.0",
```

Comment:
MInor: I'd change this to ^2.5.0. so that any other minor version (>=2.5.0) can also used, so that integrating repos, that might use cardinal won't have to install 2 versions of the same package. Wdyt?

Reply:
yea that makes perfect sense to me. just updated. btw do we have to do the same for `packages/dialect-react-ui/package.json`